sklearn/utils/linear_assignment_.py

```diff
@@ -2,12 +2,12 @@
 Solve the unique lowest-cost assignment problem using the
 Hungarian algorithm (also known as Munkres algorithm).
 
+This module is deprecated in 0.21 and will removed from 0.23
+as the algorithm incorporated in SciPy will be used instead.
+
 """
 # Based on original code by Brain Clapper, adapted to NumPy by Gael Varoquaux.
```

Comment:
please use the warnings module to raise a deprecation warning here (so that it appears whenever the module is imported)

Reply:
done